what-is-mtls.md

```diff
@@ -0,0 +1,191 @@
+# What is MTLS?
+You’ve probably heard about TLS. It is a security protocol that sits between the transport and application layers (in the TCP/IP model) and essentially encrypts application-layer traffic. This encryption provides a guarantee to the client that:
```

Comment:
Style nit: I'd remove `you probably heard about TLS` and the paragraph still reads well.